### DISCUSSION TITLE
Budget Overview Being Odd

### DISCUSSION THREAD

**Discussion by astr0n8t** [firefly-iii/firefly-iii#8504] [Firefly III]
Hi, does anyone know why my monthly auto-budgets show up like this:
<img width="494" alt="image" src="https://github.com/firefly-iii/firefly-iii/assets/13420786/653d8f2a-9f6c-4d16-8268-2821b2dec791">

They are configured with "Set a fixed amount every period", the auto budget amount is set, and the auto budget period is set to monthly.  But my budgets show like this with a duplicate for every day of the month of the amount of the month which causes it to show as (total for budget)*31 as budgeted.  This only happens on the /budgets page and they still function correctly elsewhere.

**Comment by JC5** [comment]
That's very weird! It looks like many budget amounts are created, and on the right of your screenshot are many dropdowns that allow you to set an amount, right?

**Reply by astr0n8t** [reply]
Yes, it's really odd too cause I think I've had it happen and there's only been a single dropdown and then sometimes there's one dropdown per duplicate.

**Reply by JC5** [reply]
Could you send me a full screenshot at james@firefly-iii.org?
